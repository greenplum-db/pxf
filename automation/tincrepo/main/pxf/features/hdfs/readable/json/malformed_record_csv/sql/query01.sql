-- start_matchsubs
--
-- m/DETAIL/
-- s/DETAIL/CONTEXT/
--
-- m/CONTEXT:.*line.*/
-- s/line \d* of //g
--
-- m/^CONTEXT: */
-- s/^CONTEXT: *//
--
-- m/pxf:\/\/(.*)\/pxf_automation_data/
-- s/pxf:\/\/.*\/pxf_automation_data.*tweets-broken.json\?PROFILE=.*:json/pxf:\/\/pxf_automation_data\/tweets-broken.json?PROFILE=json/
--
-- end_matchsubs
-- @description query01 for PXF HDFS Readable Json with malformed record test cases
SELECT *
FROM jsontest_malformed_record
ORDER BY id;
