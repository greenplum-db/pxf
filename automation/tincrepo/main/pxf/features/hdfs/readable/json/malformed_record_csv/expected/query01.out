-- start_matchsubs
--
-- m/DETAIL/
-- s/DETAIL/CONTEXT/
--
-- m/CONTEXT:.*line.*/
-- s/line \d* of //g
--
-- m/^CONTEXT: */
-- s/^CONTEXT: *//
--
-- m/pxf:\/\/(.*)\/pxf_automation_data/
-- s/pxf:\/\/.*\/pxf_automation_data.*tweets-broken.json\?PROFILE=.*:json/pxf:\/\/pxf_automation_data\/tweets-broken.json?PROFILE=json/
--
-- end_matchsubs
-- @description query01 for PXF HDFS Readable Json with malformed record test cases
SELECT *
FROM jsontest_malformed_record
ORDER BY id;
ERROR:  extra data after last expected column
CONTEXT:  External table jsontest_malformed_record, pxf://pxf_automation_data/tweets-broken.json?PROFILE=json&IDENTIFIER=created_at: ",,,,,,,"PXFERRMSG> error while parsing json record 'Unexpected character (':' (code 58)): was expect..."
SELECT *
FROM jsontest_malformed_record_filefrag
ORDER BY id;
ERROR:  extra data after last expected column
CONTEXT:  External table jsontest_malformed_record_filefrag, pxf://pxf_automation_data/tweets-broken.json?PROFILE=json&IDENTIFIER=created_at&SPLIT_BY_FILE=true: ",,,,,,,"PXFERRMSG> error while parsing json record 'Unexpected character (':' (code 58)): was expect..."
